tags/tags.go

```diff
@@ -75,5 +75,24 @@ func Highway() map[string]bool {
 	tags["service"] = false
 	tags["tertiary"] = false
 	tags["road"] = false
+	tags["pedestrian"] = false
+	tags["unclassified"] = false
+	tags["living_street"] = false
+	tags["track"] = false
 	return tags
 }
+
+// Just a wrapper function to print the default highway tags in the help menu
+func HighwayTagsAsString() string {
+	var highwaySlice string
+	i := 0
+	for k := range Highway() {
+		if highwaySlice == "" {
+			highwaySlice += k
+		} else {
+			highwaySlice += "," + k
+		}
+		i++
+	}
+	return highwaySlice
+}
```

Comment:
You can simplify this as such:

```go
import "strings"

func HighwayTagsAsString() string {
	  var keys []string
	  for k := range Highway() {
	      keys = append(keys, k)
	  }
	  return strings.Join(keys, ",")
}
```

Note: maps in Go have undefined iteration order, you might want to add a sort before the return in order to ensure the ordering is deterministic:

```go
import "sort"

sort.Strings(keys)
```